### PR TITLE
Fusion/feat/reskin single roadmap page

### PR DESCRIPTION
**Pull request by nesdc88**
## Ticket
https://trello.com/c/7jLph6zE/535-reskin-single-roadmap-page

## Description
RESKIN Single Roadmap Page

## What solved

- [X] Should update the Roadmap timeline
- [X] Should adjust the card responsiveness of the roadmap timeline

## Checklist:

- [X] I have performed a self-review of my own code
- [x] I have performed a self-review of my own chromatic changes
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] I have checked my code and corrected any misspellings
- [X] I have removed any unnecessary console messages
- [X] I have removed any commented code
- [x] I have checked that there are no buggy stories in Storybook

### CODE REVIEW

**Review by alejandrocabriales — APPROVED** [review]
LGTM

**Review by YasielCabrera — APPROVED** [review]

**Review by YasielCabrera — CHANGES_REQUESTED** [review]
Commented above

File: public/assets/svg/circle_outlined.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (nesdc88) [reply public/assets/svg/circle_outlined.svg]: Done

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**